src/Service.Tests/Unittests/ConfigValidationUnitTests.cs

```diff
@@ -1654,11 +1654,11 @@ string[] linkingTargetFields
             DisplayName = "GraphQL path not starting with forward slash.")]
         [DataRow("/-api", null, ApiType.GraphQL, false,
             DisplayName = "GraphQL path containing hyphen (-)")]
-        [DataRow("/api path", null, ApiType.GraphQL, false,
+        [DataRow("/api path", "GraphQL path updated Path contains white spaces.", ApiType.GraphQL, true,
```

Comment:
What is this change? Why do we need to change the existing tests?

Reply:
As explained, we would need this as we are adding a white spaces check in the URI component of the validate file